version/version.go

```diff
@@ -0,0 +1,10 @@
+package version
+
+const (
+	// Version is the Operator's version
+	Version = "0.11.0-beta.2"
+
```

Comment:
Can we factor out here something like "LatestAstarte011Version" (and LatestAstarte010Version etc)
so that we don't have to replace all instances of `0.11.0-beta.1` around?

Reply:
This is actually the version of the Operator itself. The rest is entirely handled by constraints